test/nginx-docker-compose.yaml

```
@@ -1,9 +1,7 @@
-version: "3.2"
-
 services:
   nginx:
     # https://github.com/macbre/docker-nginx-http3#quic--http3-support
-    image: ghcr.io/macbre/nginx-http3:1.27.0
+    image: ghcr.io/macbre/nginx-http3:1.29.3
```

```suggestion
    image: ghcr.io/macbre/nginx-http3:1.27.0
```

Bring back the previous nginx version.